meteor/server/publications/packageManager/expectedPackages/contentObserver.ts

```diff
@@ -0,0 +1,118 @@
+import { Meteor } from 'meteor/meteor'
+import { PartInstanceId, StudioId } from '@sofie-automation/corelib/dist/dataModel/Ids'
+import { logger } from '../../../logging'
+import {
+	ExpectedPackagesContentCache,
+	createReactiveContentCache,
+	rundownPlaylistFieldSpecifier,
+	pieceInstanceFieldsSpecifier,
+} from './contentCache'
+import { ExpectedPackages, PieceInstances, RundownPlaylists } from '../../../collections'
+import { equivalentArrays, waitForPromise } from '../../../../lib/lib'
+import { ReactiveMongoObserverGroup, ReactiveMongoObserverGroupHandle } from '../../lib/observerGroup'
+import _ from 'underscore'
+
+const REACTIVITY_DEBOUNCE = 20
```

Comment:
Should the various reactive content caches use the same/more often reuse the same value here? I expect that this may be helpful.